src/util.ts

```diff
@@ -132,4 +132,18 @@ const tags = new Set([
   // 'tspan'
 ]);
 
-export { tags };
+const validAttributes = new Set([
+  'href',
+  'class',
+  'autofocus',
+  'disabled',
+  'target',
+  'src'
+]);
+
+const reactAttributesMap = new Map<string, string>([
+  ['class', 'className'],
```

Comment:
The other special case is `for -> htmlFor`. But then there's this whole list https://reactjs.org/docs/dom-elements.html#all-supported-html-attributes, e.g. `tabindex -> tabIndex` (autoFocus belongs there as well). I'm not yet sure how that should be handled